src/components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,15 @@
+import { ErrorText } from '../../helpers/ErrorText';
+import { ErrorSpec } from '../../types/ErrorSpec';
+
+type Props = {
+  error: ErrorSpec | null;
+};
+
+export const ErrorMessage: React.FC<Props> = ({ error }) => {
+  return (
+    <span>
+      {ErrorText(error)}
+      <br />
```

Comment:
It’s better not to use this `<br />` tag
Because if you need some kind of indentation, then CSS is used for this

Reply:
it was into basic code of that task